repository-data/application/src/main/resources/hcm-content/content/documents/administration/configuration/local-feature---visitscotland.com.yaml

```
@@ -0,0 +1,202 @@
+        the list of Countries including the United Kingdom countries: Scotland, England,
+        Wales & Northern Ireland', Static file with all validation messages translated
+        into different languages, "Marketo Configuration \r\n\r\n\r\nMunchkin Id for\
+        \ Live:   638-HHZ-510\r\nMunchkin Id for Test:    830-QYE-256", "URL for Live\
+        \ : //e.visitscotland.com\r\nURL for Test: //830-QYE-256.mktoweb.com", Public
+        Recaptcha Key for Forms, '', Marketo Script required for the form to work,
+      '', '', '', '', This property is used to set up the Cludo Chatbox functionality,
+      This property is used to set up the Cludo Chatbox functionality, This property
+        is used to set up the Cludo Chatbox functionality, '', '', '', '']
+    resourcebundle:id: vs.site.config
+    resourcebundle:keys: [iknow-community.url, iknow-community.tagged-discussion,
+      links.internal-sites, links.convert-to-relative, og.site-name, og.twitter.site,
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 26 but found 6

(indentation)